satpy/tests/reader_tests/test_goes_imager_hrit.py

```diff
@@ -167,6 +171,7 @@ def suite():
     loader = unittest.TestLoader()
     mysuite = unittest.TestSuite()
     mysuite.addTest(loader.loadTestsFromTestCase(TestHRITGOESPrologueFileHandler))
+    mysuite.addTest(loader.loadTestsFromTestCase(TestHRITGOESFileHandler))
```

Comment:
Since that test was not included in the suite, the problems with byte strings were never noticed